graylog2-web-interface/src/logic/rest/FetchProvider.js

```diff
@@ -76,7 +78,7 @@ export class Builder {
 
         // Redirect to the start page if a user is logged in but not allowed to access a certain HTTP API.
         if (SessionStore.isLoggedIn() && error.status === 403) {
-          history.replace(Routes.NOTFOUND);
+          ErrorsActions.report((createUnauthorizedError(error)));
```

Comment:
Maybe we could create a factory (`createFromFetchError`?) in the `ReportedErrors` module which just takes the error and returns a proper subtype? It could look at the `status` field to find out which type to return. No need to do it in this PR though.

Optional: too many parentheses.

Reply:
For now i removed the parentheses. I will add the factory when I take care of the `MissingStreamPermissions` case.